StyleCop.Analyzers/StyleCop.Analyzers.Test/StyleCop.Analyzers.Test.csproj

```diff
@@ -30,6 +30,7 @@
     <DocumentationFile>bin\Debug\StyleCop.Analyzers.Test.xml</DocumentationFile>
     <NoWarn>1591</NoWarn>
     <CodeAnalysisRuleSet>..\StyleCop.Analyzers.ruleset</CodeAnalysisRuleSet>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
This is necessary for the tests in `TestFixedStatementAsync`. If we never allow this for our test project, we will have a hard time validating any rule that touches unsafe constructions.

Besides that, I don't really see an issue with it for a test project. What is your objection exactly?